src/components/victory-label.jsx

```diff
@@ -194,20 +195,24 @@ export default class VictoryLabel extends React.Component {
 
   render() {
     this.getCalculatedValues(this.props);
+
+    const { fontSize, fontFamily, fill } = this.style;
```

Comment:
we need to be supporting all the other possible styles like `opacity` etc, so this needs to be a lot more general